### DISCUSSION TITLE
help with Mtex

### DISCUSSION THREAD

**Discussion by SMBEN** [mtex-toolbox/mtex#1350] [Ask Anything]
Dear all
I am a beginner and I use Mtex 5.7 
I run sections, but I can't work with it, where I can't make changes like “rotate the orientation data but not the spatial data”, “ebsd data”, “calculate a list of grains”, “plot the grains”… etc, 
What can I do?

![2022-02-15_211408](https://user-images.githubusercontent.com/99767925/154157153-be5e1eaa-f9d1-4193-bbf5-e2bcec5131b7.jpg)

**Comment by ralfHielscher** [comment]
I think the best option would be to join the MTEX workshop next month ... [https://mtex-toolbox.github.io/workshops](https://mtex-toolbox.github.io/workshops)

**Reply by SMBEN** [reply]
I do, but I need a solution for now at least .
